src/core/dependency-graph.ts

```diff
@@ -84,21 +95,17 @@ export class DependencyGraph {
   /**
    * Add dependencies of the given path.
    *
-   * @param path The path (i.e. url) of a document.
+   * @param url The url of a document.
    * @param newDependencies The paths of that document's direct dependencies.
    */
   addDocument(url: string, dependencies: Iterable<string>) {
     const record = this._getRecordFor(url);
     for (const dependency of dependencies) {
       record.dependencies.add(dependency);
-      const dependantRecord = this._getRecordFor(dependency);
-      dependantRecord.dependants.add(url);
-    }
-    try {
-      record.dependenciesDeferred.resolve(record.dependencies);
```

Comment:
Earlier this had a console.log here because of a worry that this would fail in a difficult to debug way. That doesn't seem to be happening, and we removed the console log but not the silly catch and throw